Source/items.h

```diff
@@ -409,11 +411,14 @@ bool IsItemAvailable(int i);
 bool IsUniqueAvailable(int i);
 void InitItemGFX();
 void InitItems();
-void CalcPlrItemVals(int p, bool Loadgfx);
-void CalcPlrInv(int p, bool Loadgfx);
+void CalcPlrItemVals(PlayerStruct &player, bool Loadgfx);
+void CalcPlrInv(PlayerStruct &player, bool Loadgfx);
 void SetPlrHandItem(ItemStruct &item, int itemData);
 void GetPlrHandSeed(ItemStruct *h);
-void GetGoldSeed(int pnum, ItemStruct *h);
+/**
+ * @brief Set a new unique seed value on the given item
+ */
+void GetGoldSeed(PlayerStruct &player, ItemStruct &gold);

```

Comment:
The name is misleading. Shouldn't it be `SetGoldSeed`?

```suggestion
/**
 * @brief Set a new unique seed value on the given item
 */
void SetGoldSeed(PlayerStruct &player, ItemStruct &gold);
```